Documentation/ApiOverview/Database/ExpressionBuilder/Index.rst

```diff
@@ -12,7 +12,7 @@ for `WHERE` and `JOIN ON` conditions, functions like :php:`->min()` may also be
 
 It takes care of building query conditions while ensuring table and column names
 are quoted within the created expressions / SQL fragments. It is a facade to
-the actual `doctrine-dbal` `ExpressionBuilder`.
+the actual Doctrine DBAL `ExpressionBuilder`.
```

Comment:
Backticks mean textrole 'code'. So I assume this should be just: Doctrine DBAL ExpressionBuilder ("Eigenname"). I know there is often 'TYPO3 CMS' (with backticks) in the whole text. That would need to be addressed too somewhen later.

Reply:
I don't think so. Doctrine was written almost everywhere in Backticks. The whole section is full of backticks that don't refere to code...